ios/Gemfile.lock

```diff
@@ -251,4 +251,4 @@ RUBY VERSION
    ruby 2.6.5p114
 
 BUNDLED WITH
-   2.2.1
+   2.2.11
```

Comment:
Any specific reason to change this?

Reply:
Absolutely not, I didn't realize it was changed... I'm thinking reinstalling some deps in my troubleshooting process did some damage